api/src/certification/results/application/livret-scolaire-route.js

```diff
@@ -10,7 +10,7 @@ const register = async function (server) {
       method: 'GET',
       path: '/api/organizations/{uai}/certifications',
       config: {
-        auth: { strategy: 'jwt-livret-scolaire', access: { scope: ['organizations-certifications-result'] } },
+        auth: { strategy: 'jwt-application', access: { scope: ['organizations-certifications-result'] } },
```

Comment:
suggestion(non-bloquante): Juste pour être cohérent avec les autres déclarations
```suggestion
        auth: { strategy: 'jwt-application', access: { scope: 'organizations-certifications-result' } },
```